README.rst

```diff
@@ -30,13 +30,232 @@
 NVIDIA Triton Inference Server
 ==============================
 
-    **NOTE: You are currently on the r20.06-v1 branch which tracks
-    stabilization towards the next release. This branch is not usable
-    during stabilization.**
+    **Triton V1 and V2: For the 20.06 release, a legacy V1 version of
+    Triton will be released from this branch. The V1 version of Triton
+    is deprecated and no releases beyond 20.06 are planned. Going
+    forward Triton V2 will continue monthly releases as described on
+    branch** `master
+    <https://github.com/NVIDIA/triton-inference-server>`_.
     
 .. overview-begin-marker-do-not-remove
 
+NVIDIA Triton Inference Server provides a cloud inferencing solution
+optimized for NVIDIA GPUs. The server provides an inference service
+via an HTTP or GRPC endpoint, allowing remote clients to request
+inferencing for any model being managed by the server. For edge
+deployments, Triton Server is also available as a shared library with
+an API that allows the full functionality of the server to be included
+directly in an application. 
+
+What's New in 1.14.0
+--------------------
+
+* Support for the new KFServing HTTP/REST, GRPC and corresponding client 
+  libraries is released on GitHub branch r20.06 and as NGC container 20.06-py3.
+
+* Support for the legacy V1 HTTP/REST, GRPC and corresponding client libraries 
```

Comment:
I would keep only this 1 bullet. Edited to:

"Support for the...  corresponding C++ and Python client libaries."